sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -30,6 +30,15 @@ import org.apache.spark.TestUtils.{assertNotSpilled, assertSpilled}
 class JoinSuite extends QueryTest with SharedSQLContext {
   import testImplicits._
 
+  override def afterEach(): Unit = {
+    try {
+      // Clear the cache table for test cases
+      spark.sharedState.cacheManager.clearCache()
```

Comment:
Yes! It should already cover this. @heary-cao What is the reason you file this PR?